packages/angular_devkit/build_angular/package.json

```diff
@@ -23,7 +23,7 @@
     "clean-css": "4.2.1",
     "coverage-istanbul-loader": "2.0.3",
     "copy-webpack-plugin": "5.1.1",
-    "core-js": "3.2.1",
+    "core-js": "3.6.4",
```

Comment:
Can we also update this on the patch branch?

Reply:
Yes. Working on a patch variant now.